langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -42,6 +42,15 @@ type KeyType anydata;
 
 # Returns number of members of a table.
 #
+# ```ballerina
+# table<Student> students = table [{name: "Jo"}, {name: "Sam"}, {name: "Bill"}];
```

Comment:
Wouldn't this be better since `Student` type is not defined?
```suggestion
# table<record {string name;}> students = table [{name: "Jo"}, {name: "Sam"}, {name: "Bill"}];
```

Reply:
It's defined at the end?

Reply:
@sanjana do you mean to define it as an anonymous type?